script.js

```diff
@@ -0,0 +1,124 @@
+var nouns = [];
+var adjectives = [];
+var adverbs = [];
+var verbs = [];
+console.log('nouns1:',nouns);
+var story;
+
+var backupNouns = ["parent", "car", "zebra", "incense"];
+var backupAdjectives = ["blue", "lovely", "grainy", "black"];
+var backupAdverbs = ["gracefully", "foolishly", "often", "obnoxiously"];
+var backupVerbs = ["burp", "walk", "swing", "scream"];
+
+function generateMadLib() {
+	nouns = [];
+	adjectives = [];
+	adverbs = [];
+	verbs = [];
+	var addNouns = function() {
+		var userNouns = document.getElementById('nouns').value;
+		userNouns = userNouns.split(",");
+		nouns = nouns.concat(userNouns);
+		nouns.shift();
+		console.log('nouns2:',nouns);
+	};
+	var addAdjectives = function() {
+		var userAdjectives = document.getElementById('adjectives').value;
+		userAdjectives = userAdjectives.split(",");
+		adjectives = adjectives.concat(userAdjectives);
+		adjectives.shift();
+	};
+	var addAdverbs = function() {
+		var userAdverbs = document.getElementById('adverbs').value;
+		userAdverbs = userAdverbs.split(",");
+		adverbs = adverbs.concat(userAdverbs);
+		adverbs.shift();
+	};
+	var addVerbs = function() {
+		var userVerbs = document.getElementById('verbs').value;
+		userVerbs = userVerbs.split(",");
+		verbs = verbs.concat(userVerbs);
+		verbs.shift();
+	};
+	removeStory();
+	addNouns();
+	addVerbs();
+	addAdverbs();
+	addAdjectives();
+	addExtraWords();
+	var radios = document.getElementsByName('select-lib');
+	for (var i = 0; i < radios.length; i++) {
+		if (radios[i].checked) {
+			setStory(radios[i].value);
+		}
+	}
+	createStory();
+	console.log('nouns4:',nouns);
+	nouns = [];
+	verbs = [];
+	adverbs = [];
+	adjectives = [];
+	console.log('nouns4:',nouns);
+};
+function removeStory() {
```

Comment:
Were you going to do something with this function? If not it is best to remove it, again its about building good habits now. When you are done with a project your code should look beautiful to the user in the browser but also it should look beautiful from the programmers perspective.